src/components/skDashboard/styles/SocialMediaCard.styled.ts

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components'
+
+interface borderColor {
+  platform: string
+}
+interface MediaCard {
+  increase: boolean
+}
+
+export const StyledCard = styled.div<borderColor>`
```

Comment:
consider using `..<{borderColor: string}>...` instead; platform is not an intuitive name especially when its referring to the border colour

Reply:
d04ff97

Reply:
d04ff97